src/services/kuboService.ts

```diff
@@ -0,0 +1,147 @@
+import axios from 'axios';
+import sharp from 'sharp';
+import {LRUCache} from 'lru-cache';
+import { logError, logInfo } from '../utils/logger';
+
+import config from '../config/config';
+
+// const maxProfileSize = config.descriptionLength + config.imageUrlLength + config.maxNameLength + config.maxImageSizeKB * 1024;
+
+class KuboService {
+  ipfs: any;
+  profileCache = new LRUCache<string, any>({max: config.cacheMaxSize});
+  blackList = new LRUCache<string, any>({max: 100000});
+
+  constructor() {
+    logInfo('constructing KuboService');
+
+    this.initialize();
+  }
+
+  initialize = async () => {
+    logInfo('Initializing KuboService');
+    const kubo = await import('kubo-rpc-client');
+
+    this.ipfs = kubo.create(config.ipfsGateway);
```

Comment:
@verkhohliad @jaensen 
Perhaps we should pass the IPFS host configuration, but not the Gateway URL?
If we pass the gateway URL here, the health check will attempt to call `<GATEWAY_URL>/api/v0/id`, but the gateway has no such endpoint (https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-id):

```suggestion
    this.ipfs = kubo.create(config.ipfs);
```

According to the [doc](https://github.com/ipfs/js-kubo-rpc-client?tab=readme-ov-file#options), it also seems that the IPFS host endpoint is supposed to be passed here